gnome-initial-setup/gnome-initial-setup-copy-worker.c

```diff
@@ -69,18 +71,47 @@ move_file_from_homedir (GFile       *src_base,
 static void
 unlock_keyring (const gchar *initial_setup_homedir)
 {
-  gchar *file;
-  gchar *password = NULL;
+  g_autofree gchar *file = g_build_filename (initial_setup_homedir, ".config/password", NULL);
```

Comment:
```suggestion
  g_autofree gchar *file = g_build_filename (initial_setup_homedir, ".config", "password", NULL);
```

Though I appreciate this is not your bug.